etc/drupal-template.html

```diff
@@ -0,0 +1,29 @@
+<!-- These styles and scripts are added to the bottom of the page. -->
+
+<style>
+
+    /*    <<contents of main.a617e044.chunk.css>>8/
```

Comment:
Can you convert this comment to an html comment with <!-- --> surrounding it

Reply:
Done. @j-tafoya please see the latest commit.

Reply:
You rock! Thank you!